### PR TITLE
Mention GSOC13 on front page

### PR DESCRIPTION
**Pull request by astrofrog**
@eteq - what do you think?

### CODE REVIEW

**Comment by eteq** [comment]
Sure, seems fine to me - your plan is to take it down again once the student window has closed, right?

**Comment by astrofrog** [comment]
Yep!

File: index.rst [hunk 1]
```diff
@@ -38,6 +38,13 @@ questions of all sorts!  If you want to get involved in Astropy development
 efforts, or other  more technical discussions of Astropy, join the 
 `astropy-dev`_ list.
 
+**Google Summer of Code 2013**: Astropy is officially a project for the Google
```
Comment (eteq) [inline index.rst:41]: Maybe rephrase this first part as something like "Astropy is accepting students for the Google Summer of Code 2013 ..."? Then it's more obvious what we want.

**Comment by eteq** [comment]
Other than that possible phrasing change, feel free to merge if you like it (and push up to the site).

**Comment by eteq** [comment]
A bit late, but I'll go ahead and do this now along side some of the other PRs
